Formula/pldiff.rb

```diff
@@ -0,0 +1,16 @@
+class Pldiff < Formula
+  desc "Compares plist files line by line"
+  homepage "https://github.com/scottrigby/pldiff"
+  url "https://github.com/scottrigby/pldiff/archive/1.0.4.tar.gz"
+  sha256 "0093027aee4ceb049902a9fcea28923477c87c2f88813e2370fb4c12195b9ccd"
+
+  depends_on "colordiff" => :recommended
+
+  def install
+    system "/bin/sh", "install.sh", prefix
+  end
+
+  test do
+    system "false"
```

Comment:
@woodruffw Understood, I just didn't add this yet. I would think for this command, the test would make two temp files (using `EOF`) with one minor change, then call the `pldiff` command and then measure expected output from the CLI. Does that sound reasonable?